mappings/net/minecraft/entity/vehicle/AbstractMinecartEntity.mapping

```diff
@@ -34,7 +35,7 @@ CLASS axm net/minecraft/entity/vehicle/AbstractMinecartEntity
 	METHOD d setDamageWobbleSide (I)V
 		ARG 1 wobbleSide
 	METHOD k setCustomBlockOffset (I)V
-	METHOD l getDamageWobbleStrength ()F
+	METHOD l getDamageWobbleMax ()F
```

Comment:
What does this fix? I was asking if the strength is measured as an angle. If yes, then `getDamageWobbleAngle` would be better. Otherwise, `getDamageWobbleStrength` is ok.

Reply:
The problem is that your suggestion is (IMO) wrong and you (justifiably) do not like my initial naming. I attempted to compromise by making it clear that we're working in angles by naming the argument of `setDamageWobbleMax` `maxAngle`. If you want more than that, `wobble` can be globally replaced with `wobbleAngle` (I think it would be inconsistent to only change it for just the two methods you pointed out), but I have not done so yet because, IMO, that would be very verbose.

Reply:
> The problem is that your suggestion is (IMO) wrong

How is it wrong, is it not an angle?

> you (justifiably) do not like my initial naming

If a more descriptive name exists (`getDamageWobbleAngle`), it should be named that, rather than forcing people who want to use it to search through the code to figure out what the unit is.

> making it clear that we're working in angles by naming the argument of setDamageWobbleMax

I really don't see how adding `max` make it clear what the unit is?

>  If you want more than that, wobble can be globally replaced with wobbleAngle (I think it would be inconsistent to only change it for just the two methods you pointed out)

There are no names involving `wobble` other than those you just named...

Reply:
`getDamageWobbleMax` is definitely wrong; is it just `getMaxWobbleDamage`, where `damage` is the actual unit?

Reply:
@liach Based on the current name, I'm assuming this is the angle by which the minecart wobbles when damaged.

Reply:
`getWobbleRoll`

Reply:
The units are degrees, yes. Other names with `wobble` that we may want to replace with `wobbleAngle` in them:

* `DAMAGE_WOBBLE_MAX`, `getDamageWobbleMax`, `setDamageWobbleMax` (these appear in twice, in `AbstractMinecartEntity` and `BoatEntity`)
* `BUBBLE_WOBBLE_MAX`, `getBubbleWobbleMax`, `setDamageWobbleMax`
* `lastBubbleWobble`, `bubbleWobble`, `interpolateBubbleWobble`

The damage wobble is not a roll (side-to-side), it is a pitch (front-to-back). I don't know of a name for the bubble wobble; boats wobble sort of diagonally, roll and pitch, while over bubble columns. Also, in `AbstractMinecartEntity`, this is not actually damage wobble, just wobble, as it wobbles while hitting activator rails, too. I didn't see that the first time around.

Now, (as I've said before), *any* name like `getWobbleRoll` or `getWobbleAngle` is inaccurate. This variable is only the maximum allowed wobble (as an angle). The *actual* wobble, for the minecart and for the boat's damage wobble, is only calculated in the renderer, incorporating a `sin` function and the world time. The boat's `bubbleWobble` field holds the calculated wobble, which is updated every `tick()`, and the `interpolateBubbleWobble` method is what actually gets the rendered angle. I am OK with naming `interpolateBubbleWobble` as such, because it actually tells you the wobble. I am not OK with naming `getDamageWobbleMax` `getWobble` etc., because it doesn't.

If it's going to be causing this much confusion, then I guess it's best to just be as explicit as possible. These six data tracker methods should be named `{g,s}et[Reason]WobbleMaxAngle`, the data trackers `[REASON_]WOBBLE_MAX_ANGLE`, and the three stragglers in `BoatEntity` should be `[last]BubbleWobbleAngle` and `interpolateBubbleWobbleAngle`.

Reply:
> Other names with wobble that we may want to replace with wobbleAngle in them

You named all of these. My suggestion of using "wobble angle" instead of wobble applies to all your names from your last PR (except wobbleSide).

>  I am not OK with naming getDamageWobbleMax getWobble etc., because it doesn't.

When did I ever suggest `getWobble`? I am suggesting `getDamageWobbleAngle` (or `MaxAngle` if you prefer...

> If it's going to be causing this much confusion, then I guess it's best to just be as explicit as possible. These six data tracker methods should be named {g,s}et[Reason]WobbleMaxAngle, the data trackers [REASON_]WOBBLE_MAX_ANGLE, and the three stragglers in BoatEntity should be [last]BubbleWobbleAngle and interpolateBubbleWobbleAngle.

I don't understand what you're saying. Adding the word `angle` to the name was my original suggestion, which you argue against, and you then suggest the exact same thing.

Reply:
For nine of the listed names, `maxAngle` is correct, but `angle` is wrong (to me, it would imply that it's the *current* angle, which it isn't). As I read it, you suggested just `angle`, which I argued against, because, again, just `angle` is not correct. Also, I thought it would be too verbose, as all 12 names listed above would have to contain `wobbleAngle` instead of `wobble`, for consistency, which is without factoring in that 9 of them also need a `max` tacked on.  I said that we *could* use those verbose names, but, when creating this PR, I chose different names that I hoped would do instead. Now, I'm realizing that this is more trouble than it's worth, and I'm just going to use the most verbose option because we've been talking past each other for a whole day now.

> When did I ever suggest `getWobble`? I am suggesting `getDamageWobbleAngle`

I was referring to the absence of the word `Max` in the names, which is what makes the difference between correct and incorrect. `getDamageWobbleAngle` is just as incorrect as `getWobble`, without the `Max` in it. The clarification `Angle` and the qualifier `Damage` (where it's correct, sorry about that) are useful, of course.